### ISSUE TITLE
Did you publish npm modules?

### ISSUE THREAD

**Issue by mapmeld** [readium/readium-js#139]
readium-js is registered on NPM, but the repos are all empty with version 0.0.0

https://www.npmjs.com/~readium

**Comment by danielweck** [comment]
Correct, there are placeholders for Readium build artifacts (names reserved at the NPM repository), but the pre-built RequireJS module bundles are ; at this stage ; not very useful on their own, and right now Readium's semantic versioning and release strategy isn't designed to fit in nicely with an NPM -driven build workflow (in other words, Readium apps are usually composed from GitHub source tree, at particular commit hash / release tag, etc. ).
There are currently no concrete plans to populate the NPM placeholders. Do you have a concrete need?

**Comment by mapmeld** [comment]
I'm thinking about using readium-js on the server so we can split an eBook up into pages and strings for translation, rather than downloading the full book and library to the mobile client.

It doesn't need to be on NPM, but if not then the readme section about NPM could be changed to reflect this and not recommend ```npm install readium-js```

**Comment by danielweck** [comment]
Oh, thank you for pointing this out, we'll update the READMEs!

**Comment by Khrystyna** [comment]
Hi! I tried to install Readium-JS via npm, but I got errors. How I actually can run any example of Readium JS?

**Comment by danielweck** [comment]
Hello @khrystyna 
Follow this link to see the Readium "cloud reader" app in action:
https://readium.firebaseapp.com
The above URL is where the latest version of the app is deployed, built from the develop branch:
https://github.com/readium/readium-js-viewer/tree/develop
The README contains all the information you need to get started (you can also build your own version of the Chrome extension).
I hope this helps.
PS: as mentioned in this discussion thread, the NPM packages are placeholders, they do not contain actual usable software components, let alone ready-to-use apps.

**Comment by Khrystyna** [comment]
@danielweck  Thank you! I will try your way 👍
